### PR TITLE
complex-numbers: check test version and update

### PR DESCRIPTION
**Pull request by trangttt**
#993

### CODE REVIEW

**Comment by m-a-ge** [comment]
@trangttt thanks for the PR
build failed due to the following:
```text
3.06s$ flake8
./exercises/complex-numbers/complex_numbers_test.py:9:1: E302 expected 2 blank lines, found 1

The command "flake8" failed and exited with 1 during .
Your build has been stopped.
```

File: exercises/complex-numbers/complex_numbers_test.py [hunk 1]
```diff
@@ -126,13 +115,37 @@ def test_conjugate_a_purely_imaginary_number(self):
...
+
+    def test_imaginary_part_of_a_purely_real_number(self):
+        input_number = ComplexNumber(1, 0)
+        self.assertEqual(input_number.imaginary, 0)
+
+    def test_imaginary_part_of_a_purely_imaginary_number(self):
+        input_number = ComplexNumber(0, 1)
+        self.assertEqual(input_number.imaginary, 1)
+
+    def test_maginary_part_of_a_number_with_real_and_imaginary_part(self):
```
Comment (cmccandless) [inline exercises/complex-numbers/complex_numbers_test.py]: Spelling error that wasn't caught in the previous version of this file. Should be
```Python
def test_imaginary_part_of_a_number_with_real_and_imaginary_part(self):
```

File: exercises/complex-numbers/complex_numbers_test.py [hunk 2]
```diff
@@ -4,74 +4,64 @@
 
 from complex_numbers import ComplexNumber
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```
Comment (N-Parsons) [inline exercises/complex-numbers/complex_numbers_test.py]: Can you move the comment down by a line for the sake of consistency? (ie. two blank rows before, one after)

File: exercises/complex-numbers/complex_numbers_test.py [hunk 3]
```diff
@@ -126,13 +116,37 @@ def test_conjugate_a_purely_imaginary_number(self):
         self.assertEqual(expected.imaginary,
                          input_number.conjugate().imaginary)
 
-    def conjugate_a_number_with_real_and_imaginary_part(self):
+    def test_conjugate_a_number_with_real_and_imaginary_part(self):
         input_number = ComplexNumber(1, 1)
         expected = ComplexNumber(1, -1)
         self.assertEqual(expected.real, input_number.conjugate().real)
         self.assertEqual(expected.imaginary,
                          input_number.conjugate().imaginary)
```
Comment (N-Parsons) [inline exercises/complex-numbers/complex_numbers_test.py]: Can you swap the arguments in these asserts? They should follow the convention of `assertEqual(actual, expected)`. This should also be done for the division tests above.

**Comment by N-Parsons** [comment]
@trangttt, are you still working on this? It just needs a few minor changes and then it can be merged.

File: exercises/complex-numbers/complex_numbers_test.py [hunk 4]
```diff
@@ -5,73 +5,64 @@
...
         first_input = ComplexNumber(1, 2)
-        second_input = ComplexNumber(-5, 10)
-        self.assertEqual(first_input.mul(second_input).real, -25)
-        self.assertEqual(first_input.mul(second_input).imaginary, 0)
+        second_input = ComplexNumber(3, 4)
+        self.assertEqual(first_input.mul(second_input).real, -5)
+        self.assertEqual(first_input.mul(second_input).imaginary, 10)
 
     def test_divide_purely_real_numbers(self):
         input_number = ComplexNumber(1.0, 0.0)
```
Comment (N-Parsons) [inline exercises/complex-numbers/complex_numbers_test.py]: Can you swap the order the arguments in these methods to match the convention `assertEqual(actual, expected)`?

**Comment by N-Parsons** [comment]
@trangttt, can you swap the argument order in the division tests to match the `assertEqual(actual, expected)` convention? Once that's done, this PR will be ready to merge.

**Comment by N-Parsons** [comment]
Perfect! Thanks, @trangttt!

**Pull request merged**